packages/keyda-demo/src/lib/FormToSubmit/index.js

```diff
@@ -0,0 +1,48 @@
+import React, { useCallback } from 'react';
+import _ from 'lodash';
+import axios from 'axios';
+
+import { useKeyStateState } from '../Context';
+
+const FormToSubmit = (props) => {
+  const definedFormTypes = ['REGISTER', 'LOGIN', 'PASSWORD_CHANGE'];
+
+  const { onSubmit, formType, children, ...rest } = props;
+
+  if (!formType) {
+    throw new Error(
+      'You should pass the "formType" prop explicitly to define a mechanism of KeyDa form.\nYou can choose one of "REGISTER", "LOGIN", "PASSWORD_CHANGE" matched with your usage.'
+    );
+  } else if (!_.includes(definedFormTypes, formType)) {
+    throw new Error(
+      'You passed wrong "formType". You can choose one of "REGISTER", "LOGIN", "PASSWORD_CHANGE" matched with your usage.'
+    );
+  }
```

Comment:
To use this form in a diverse way, I made the types of the forms, and if `formType` prop is missing, it will throw the error to the user.